lib/galaxy/schema/schema.py

```diff
@@ -2068,9 +2068,9 @@ class SubworkflowStep(WorkflowStepBase):
     )
 
 
-class Creator(Model):
-    class_: str = Field(..., alias="class", title="Class", description="The class representing this creator.")
-    name: str = Field(..., title="Name", description="The name of the creator.")
+class Thing(Model):
```

Comment:
By intuition this looks stange to me. Before `Creator` had an address etc. And now it hasn't, or?

Reply:
I extended creator to inherit from thing, so I only moved the attributes into a common base class - maybe this shows up weirdly in the GitHub UI?

Reply:
I only see `class Creator(Model):`, should that have been `class Creator(Thing):` ?

Reply:
True, didn't restore the inheritance.